shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -732,6 +773,16 @@ public static TextDirection fromInt(int value) {
         }
     }
 
+    private class CustomAccessibilityAction {
+        CustomAccessibilityAction() {}
+
+        int resourceId = -1;
```

Comment:
Maybe add a comment here to explain to future-me what the difference between resourceId and id is? And maybe also what the label is used for?

Reply:
Done